### ISSUE TITLE
Hyperlink the tissues in the model and gene profile pages

### ISSUE THREAD

**Issue by emepyc** [emepyc/project-score#9]
Right now it is not easy to go from the model profile page to the tissue page (table page with the tissue set).
It would be nice to have the tissues hyperlinked to allow that navigation.

**Comment by emepyc** [comment]
Since we are going to filter by analyses / cancer type I think we need to include a new column in the table with the cancer type alongside the tissue

**Comment by emepyc** [comment]
depends on #19

**Comment by Donnyvdm** [comment]
I don't think this needs to be in the table, just a link in the top left would suffice. See example mocks below
1. 
![Screen Shot 2019-08-20 at 15 54 53](https://user-images.githubusercontent.com/4202651/63358711-69758980-c363-11e9-895b-faecf7cc48b0.png)
2. 
![Screen Shot 2019-08-20 at 15 56 37](https://user-images.githubusercontent.com/4202651/63358714-6b3f4d00-c363-11e9-95c0-164e3838e05c.png)

**Comment by emepyc** [comment]
Ah, great. That will work. Thanks!
